packages/lesswrong/components/localGroups/LocalGroupPage.tsx

```diff
@@ -79,13 +84,13 @@ const LocalGroupPage = ({ classes, documentId: groupId }: {
 
   return (
     <div className={classes.root}>
-      {group.googleLocation && <CommunityMapWrapper
+      {group.googleLocation ? <CommunityMapWrapper
         terms={{view: "events", groupId: groupId}}
         groupQueryTerms={{view: "single", groupId: groupId}}
         mapOptions={{zoom:11, center: group.googleLocation.geometry.location, initialOpenWindows:[groupId]}}
-      />}
+      /> : <div className={classes.bannerImage}></div>}
```

Comment:
I'm working on adding a banner image to this page anyway (like the one for sequences), so I'm going to add this placeholder here to fix the margin issues I'm seeing.